draft-ietf-rats-corim.md

```diff
@@ -1249,6 +1253,26 @@ applies to all measurements in the triple, including those in `measurement-value
 {::include cddl/conditional-endorsement-triple-record.cddl}
 ~~~
 
+#### Multi Conditional Endorsement Triple {#sec-comid-triple-multi-cond-end}
+
+A Multi Conditional Endorsement Triple behaves similarly to the Conditional Endorsement Triple {{sec-comid-triple-cond-end})
```

Comment:
I think the first statement is not strictly true if you revisit the definition of conditional-endorsement-triple-record. There the First reference value matching stop the matching process. However in this triple, we expect to match All the Subject conditions prior to accepting the series.